Character/Classes/Sorcerer/Sorcerer.xml

```diff
@@ -147,19 +147,21 @@
 				<text>At 3rd level, you gain the ability to twist your spells to suit your needs. You gain two of the following Metamagic options of your choice. You gain another one at 10th and 17th level.</text>
 				<text>	You can use only one Metamagic option on a spell when you cast it, unless otherwise noted.</text>
 			</feature>
+		</autolevel>
+		<autolevel level="3" IL="IL">
```

Comment:
This doesn't seem to actually change anything

Reply:
That separates Metamagic options and marks them as Inlined List, so that user can exclude them with IL option